### PR TITLE
FIX: Darkness

### PR DESCRIPTION
**Pull request by larsoner**
Before:
Screen Shot 2022-03-04 at 2 00 45 PM

After:
Screen Shot 2022-03-04 at 2 01 00 PM

### CODE REVIEW

**Comment by larsoner** [comment]
And on Linux, before:
Screenshot from 2022-03-04 14-21-22

After:

Screenshot from 2022-03-04 14-25-37

FYI this requires `darkdetect` and `qdarkstyle`

**Comment by larsoner** [comment]
@hoechenberger now that https://github.com/mne-tools/mne-python/pull/10414 is merged in MNE-Python, you should be able to test here. You should end up with a slightly blue-ish gray instead of plain gray if things are working properly, just like in Brain.

**Comment by larsoner** [comment]
@marsipu this is ready for review/merge from my end now!

File: mne_qt_browser/_pg_figure.py [hunk 1]
```diff
@@ -2598,17 +2598,26 @@ class PyQtGraphBrowser(BrowserBase, QMainWindow, metaclass=_PGMetaClass):
 
     gotClosed = pyqtSignal()
 
-    def __init__(self, splash=None, **kwargs):
+    def __init__(self, **kwargs):
         self.backend_name = 'pyqtgraph'
 
         BrowserBase.__init__(self, **kwargs)
         QMainWindow.__init__(self)
-        self._splash = splash
```
Comment (larsoner) [inline mne_qt_browser/_pg_figure.py:2606]: @marsipu I didn't realize that the `kwargs` ended up being in `self.mne.<kwargs>`. I simplified a bit of the code here based on this.

Comment (marsipu) [reply mne_qt_browser/_pg_figure.py:2606]: Nice:+1:

**Review by marsipu — APPROVED** [review]

**Comment by marsipu** [comment]
That looks great, thank you @larsoner

**Pull request merged**